docs/docs/pages/basics/fixtures.mdx

```diff
@@ -0,0 +1,73 @@
+import { RefLink } from "../../components/RefLink";
+
+# Fixtures [Static test assets for deterministic scenarios]
+
+A _fixture_ is any static asset—image, audio clip, JSON payload—that your scenario test loads at runtime to ensure **repeatable, offline-friendly** execution.
+
+:::tip
+**Why use fixtures?**
+
+- **Determinism**: The exact same bytes are fed to the agent and judge on every run.
+- **Speed & cost**: No network fetches or external dependencies.
+- **Debugging**: Easy to reproduce failures locally.
+  :::
+
+## Where to put fixtures
+
+- Create a `fixtures/` directory **next to your test file**.
+- Commit the asset (e.g. `scenario.webp`) to version control so CI has it.
+- Keep binary files small—**&lt;100&nbsp;KB for images** is a good rule of thumb.
+
+```text
+javascript/examples/vitest/tests/
+└─ fixtures/
+   └─ scenario.webp
+```
+
+## Loading a fixture in TypeScript
+
+```ts
+import * as fs from "fs";
+import * as path from "path";
+
+function getImageDataURL(filename: string): string {
+  const filePath = path.join(__dirname, "fixtures", filename);
+  const buffer = fs.readFileSync(filePath);
+  return `data:image/webp;base64,${buffer.toString("base64")}`;
```

Comment:
I can already envision people copy and pasting this and surprised jpegs don't work. Maybe we pass filetype as a second argument instead of hardcoding only `image/webp`?